packages/manager/modules/iam/src/components/policyIdentities/policyIdentities.controller.js

```diff
@@ -0,0 +1,106 @@
+import { ROUTES } from '@iam/routes';
+import { decodeUrn, encodeUrn } from '@iam/resolves';
+import { URN_VERSION, ENTITY } from '@iam/constants';
+
+export default class PolicyIdentitiesController {
+  /* @ngInject */
+  constructor($q, $translate, IdentityService, PolicyService, coreConfig) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.IdentityService = IdentityService;
+    this.PolicyService = PolicyService;
+    this.region = coreConfig.getRegion();
+    this.user = coreConfig.getUser();
+  }
+
+  $onInit() {
+    this.loading = true;
+
+    this.identities = this.policy.identities
+      .map(decodeUrn)
+      .map((urn) => ({ title: urn.components.slice(-1).toString(), urn }));
```

Comment:
The last component will always be a string, is `toString()` needed ?

Reply:
It's applied =>  [...urn.components].pop()